Update issue and PR templates (#658)
devicons/devicon: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug Report

> about: Report a bug/issue that you found in the SVGs, CSS files, etc...

Reviewer: ```suggestion
about: Report a bug/issue that you found in the SVGs, CSS files, etc..
```

Author: I was curious about and search it up. Turns out we only need 1 period after etc.

title: 'Bug Report: [NAME]'
labels: 'bug'
assignees: ''

---

### Bug

> *Short description about the bug that you found. Provides images/code if applicable.*

Reviewer: ```suggestion
*Short description of the bug that you found. Provide images/code if applicable.*
```

### How to replicate the bug

> *What should we do to find this bug*

Reviewer: ```suggestion
*What should we do to replicate this bug.*
```
